src/java_bytecode/java_object_factory.cpp

```diff
@@ -494,6 +495,141 @@ class recursion_set_entryt
   }
 };
 
+/// Get max value for an integral type
+/// \param type:
+///   Type to find maximum value for
+/// \return Maximum integral valu
+static size_t max_value(const typet& type)
+{
+  if(type.id()==ID_signedbv)
```

Comment:
Check this is really an int32!

Reply:
Can't do. It breaks half of all regression tests.